frontend/packages/topology/src/actions/provider.ts

```diff
@@ -17,3 +26,26 @@ export const useTopologyWorloadActionProvider = (element: GraphElement) => {
     return [actions, true, undefined];
   }, [actions, element]);
 };
+
+export const useTopologyEdgeActionProvider = (element: BaseEdge) => {
```

Comment:
Had to add this hook here because importing `edgeActions` in the knative-plugin providers increases the main vendor bundle size